pkg/exporter/sumologicexporter/README.md

```diff
@@ -58,6 +58,13 @@ exporters:
     # source templating is going to be applied,
     # default = `%{_metric_}`
     graphite_template: <graphite_template>
+    # name of resource attribute which should be dropped for records
+    # this is for attribute used by routing processor
+    # other attributes should be removed by processors in pipelines before
+    # This is workaround for the following issue:
+    # https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/7407
+    # default = `sumo_routing`
+    drop_routing_atttribute: <drop_routing_atttribute>
```

Comment:
The name is a bit unfortunate IMHO. It indicates that it should be a bool.

Reply:
would `routing_attribute_to_drop` be a better name?

Reply:
Not sure about either of those 😬 . But let's go with your proposal: `routing_attribute_to_drop`

Reply:
fixed then :shrug: